<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mapping</class>
 <widget class="QWidget" name="Mapping">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>463</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Visual Mapping</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Permissions</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dateAnCheckBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Triad</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="sizeShowCheckBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="permissionAnCheckBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="typeShowCheckBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>55</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Arc</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string> Pie</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="timeShowCheckBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="sizeAnCheckBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>55</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Spiral</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Attributes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>55</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Primary</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Secondary</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Primary</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="dateComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Modified</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Read</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Created</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="dateShowCheckBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>140</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Glyph</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Animation</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colormapButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Colour Map</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="permissionShowCheckBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="typeAnCheckBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="timeAnCheckBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>21</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="colormapButton_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Mapping</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="customGroupBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>431</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Custom Transfer Function</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="TransferFunction" name="sizeTF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>410</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>410</width>
      <height>110</height>
     </size>
    </property>
   </widget>
   <widget class="TransferFunction" name="dateTF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>410</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>410</width>
      <height>110</height>
     </size>
    </property>
   </widget>
   <widget class="TransferFunction" name="timeTF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>410</width>
      <height>110</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>410</width>
      <height>110</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Size (Scale TF)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>150</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Date (Length TF)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Time (Length TF)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="customDurationSpinBox">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>0</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1500.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1000.000000000000000</double>
    </property>
    <property name="value">
     <double>3000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(ms)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="defaultGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>321</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Default Transfer Function</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Duration </string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="defaultDurationSpinBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>200.000000000000000</double>
    </property>
    <property name="value">
     <double>6000.000000000000000</double>
    </property>
   </widget>
   <widget class="QComboBox" name="defaultTFComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>78</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Function#</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>(ms)</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>430</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>430</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>430</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QSlider" name="opacitySlider">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Opacity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="opacityLineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>290</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.0</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TransferFunction</class>
   <extends>QGraphicsView</extends>
   <header>TransferFunction.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
